h2o-algos/src/main/java/hex/gam/GAM.java

```diff
@@ -384,10 +439,11 @@ void verifyGamTransformedFrame(Frame gamTransformed) {
     
     @Override
     public void computeImpl() {
-      init(true);     //this can change the seed if it was set to -1
+      if (_doInit)                  // disable when in CV and building the main model
+        init(true);       //this can change the seed if it was set to -1
+      validateGamParameters();
```

Comment:
I wonder why do we need this, however, if we keep `_doInit` then we should do

if (_doInit) {
 init(true)
} else {
 validateGamParameters();
}

however, skipping init is suspicious

Reply:
Thank you for catching this. I though this is there because of the way GAM is instantiated, but it seems to be false theory. I'll look into it.

Reply:
The only things that seems to be really effectively required from init is the `validateGamParameters` anyway.